### PR TITLE
(Sentinel) Migrate sentinel codebase (root -> node)

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

- removal of pre-existing code will be done after it has been confirmed to work as expected

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

**Comment by nick-bisonai** [comment]
@coderabbitai resolve

File: node/cmd/sentinel/main.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	"github.com/rs/zerolog/log"
+)
+
+func main() {
+	ctx := context.Background()
+	logLevel := os.Getenv("LOG_LEVEL")
+	if logLevel == "" {
+		logLevel = "info"
+	}
+	zerolog.SetGlobalLevel(getLogLevel(logLevel))
```
Comment (Intizar-T) [inline node/cmd/sentinel/main.go:26]: sentinel can use logscribe now, should we integrate it in this PR or a separate one? 😄

**Review by Intizar-T — APPROVED** [review]
lgtm!

File: node/cmd/sentinel/main.go (hunk 1 above)
Comment (nick-bisonai) [reply node/cmd/sentinel/main.go:26]: let's fully focus on just moving the codebase here 😅 integration can be done after

**Pull request merged**